modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -240,20 +242,18 @@ public void testCanReconnectAfterFail() throws Exception {
         ConnectionManagerWrapper manager2 = startManager(port2);
 
         NettySender sender = manager1.openChannelTo(manager2).get(3, TimeUnit.SECONDS);
+        assertNotNull(sender);
 
         TestMessage testMessage = messageFactory.testMessage().msg(msgText).build();
 
         manager2.close();
 
         NettySender finalSender = sender;
 
-        assertThrows(ClosedChannelException.class, () -> {
-            try {
-                finalSender.send(new OutNetworkObject(testMessage, emptyList())).get(3, TimeUnit.SECONDS);
-            } catch (ExecutionException e) {
-                throw e.getCause();
-            }
-        });
+        assertThrowsWithCause(
```

Comment:
Sorry, I just noticed that `send` returns a Future, maybe we should use `willThrowWithCauseOrSuppressed` instead?